From: =?utf-8?q?Tim_R=C3=BChsen?= <tim.ruehsen@gmx.de>
Date: Mon, 12 Dec 2016 20:56:51 +0100
Subject: Patch SO version to 6:0:1

This was discussed on the libpsl mailing list; the age should not have
been set to 0 because no existing interfaces were removed or changed
in an API-breaking way.

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)


diff --git a/configure.ac b/configure.ac
index 2ad1d45..06d64e7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -88,7 +88,7 @@ PKG_PROG_PKG_CONFIG
 # 4. If any interfaces have been added, removed, or changed since the last update, increment current, and set revision to 0.
 # 5. If any interfaces have been added since the last public release, then increment age.
 # 6. If any existing interfaces have been removed or changed since the last public release, then set age to 0. 
-AC_SUBST([LIBPSL_SO_VERSION], [6:0:0])
+AC_SUBST([LIBPSL_SO_VERSION], [6:0:1])
 AC_SUBST([LIBPSL_VERSION], $VERSION)
 
 # Check for enable/disable builtin PSL data
